File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans=0
        cnt=0
        for i in range(len(nums)):
            if nums[i]==1:
                cnt=cnt+1
            else:
                ans=max(ans,cnt)
                cnt=0
        ans=max(ans,cnt)
        return ans

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {

        int maximum=0;
        int count=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]==1){
                count+=1;
            }else{
                count=0;
            }
            maximum=Math.max(maximum,count);
        }
        return maximum;
        
    }
}